GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1069,7 +1090,8 @@ int SearchRevision(ObjectId objectId)
                     return exactIndex;
                 }
 
-                // Not found, so search for its parents
+                /* The code below is extremely slow, TryGetParents will return a huge list of parents
```

Comment:
What is the consequence of this?
Any risk to miss parents now?

Reply:
No/yes -> This causes the related commits in the file history not to be drawn in a color. All lines in the graph are grey if the top commit is not the current commit. This is indeed regression; but the loading of the history, in my case, changes from 30sec to 1sec. Not sure what to do with this.

Reply:
This could be an option (even if there are too many options already). 
I suggest this option is enabled by default though, listed as a tweak with certain effects on a wiki page or so.